setup.py

```diff
@@ -56,8 +56,8 @@ def read(*names, **kwargs):
         "wheel",
         "twine",
         # For scientific data processing and visualisation
-        "numpy",
-        "scipy",
+        "numpy<2",
+        "scipy<2",
```

Comment:
the latest version of scipy is still 1.14

Reply:
I suppose SciPy will come with an `2.0` soonish as well, following `NumPy`. Just making it future proof.

Reply:
to be fixed later